packages/udaru-core/README.md

```diff
@@ -36,6 +36,35 @@ udaru.organizations.list({}, (err, orgs) => {
 
 ```
 
+## Hooks
+
+Hooks are registered using the `udaru.addHook` method and allow you to listen to specific events in udaru life.
+
+Each udaru method exposes a namespaced hook (e.g.: the `udaru.authorize.isUserAuthorized` method exposes the `authorize:isUserAuthorized` hook).
+
+The hook is a node-style callback with three arguments: the method arguments, the method result values and a callback to invoke once done
+
+Simple example taken from [examples/hooks.js](examples/hooks.js):
+
+```
+const udaru = require('@nearform/udaru')()
+
+udaru.addHook('authorize:isUserAuthorized', function (error, args, result, done) {
+  if (error) {
+    console.error(`Authorization errored: ${error}`)
```

Comment:
so this is the contentious bit, IMO this should be:
```
if (error) {
  return done(error)
}
```

Reply:
You're right. This was a legacy of first draft when the errors were swallowed. Updating it.